train.py

```diff
@@ -35,25 +52,28 @@ def main(args):
     env.close()
 
     # Policy
-    policy = get_policy_for_env(env,
-                                hidden_sizes=config['hidden-sizes'],
-                                nonlinearity=config['nonlinearity'])
-    policy.share_memory()
+    # policy = get_policy_for_env(env,
+    #                            hidden_sizes=config['hidden-sizes'],
+    #                            nonlinearity=config['nonlinearity'])
+    # --replaced by agent
+
+    agent.share_memory() #policy.share_memory()
 
     # Baseline
-    baseline = LinearFeatureBaseline(get_input_size(env))
+    baseline = LinearFeatureBaseline(get_input_size(env)) ## input_size can agent.block_hidden_dim
```

Comment:
This will need to be changed accordingly to take as input the same type of observation as our agent.

Reply:
True.